deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -154,11 +154,25 @@ class ChatViewController: UITableViewController {
     }()
 
     private lazy var contextMenu: ContextMenuProvider = {
-        let copyItem = ContextMenuProvider.ContextMenuItem(
-        title: String.localized("global_menu_edit_copy_desktop"),
-        imageName: "doc.on.doc",
-        action: #selector(BaseMessageCell.messageCopy),
-        onPerform: { [weak self] indexPath in
+        let dcChat = dcContext.getChat(chatId: chatId)
+        let config = ContextMenuProvider()
+        if #available(iOS 13.0, *), dcChat.canSend {
+            let mainMenu = ContextMenuProvider.ContextMenuItem(submenuitems: [replyItem, forwardItem, infoItem, copyItem, replyPrivatelyItem, deleteItem])
+            config.setMenu([mainMenu, selectMoreItem])
```

Comment:
maybe take the chance to streamline this code - or is there a reason to use `mainMenu` and then anyway add another item on `setMenu`? the other branches are more straight forward,

Reply:
yes, we do this on purpose to get the space between the upper menu items and selectMore